organization/components/timeslot/TurfCard.tsx

```
@@ -0,0 +1,92 @@
+        </div>
+      </div>
+
+      {/* Selection Indicator */}
+      {isSelected && (
+        <div className="absolute top-2 right-2 bg-blue-500 text-white rounded-full p-1">
+          <svg
+            xmlns="http://www.w3.org/2000/svg"
+            className="h-5 w-5"
+            viewBox="0 0 20 20"
+            fill="currentColor"
+          >
```

Biome (1.9.4)

[error] Alternative text title element cannot be empty

For accessibility purposes, SVGs should have an alternative text, provided via title element. If the svg element has role="img", you should add the aria-label or aria-labelledby attribute.

(lint/a11y/noSvgWithoutTitle)